apps/academy/src/components/Header.tsx

```diff
@@ -36,42 +41,44 @@ const PageHeader: FunctionComponent = () => {
   const { pathname } = router;
 
   return (
-    <header className="app-container absolute left-0 right-0 top-0 z-50 flex items-start justify-between px-8 pt-8 md:flex-row">
-      <div className="hidden lg:flex ">
+    <header className=" mx-[30px] flex items-center justify-between pt-[30px] md:mx-[45px] md:pt-[20px] ">
+      <div className="hidden  lg:block">
         {pathname === "/" || pathname === "/tracks" || pathname === "/fundamentals" ? (
-          <div className="nav-division flex flex-col gap-y-6">
-            <div className="flex justify-between">
-              <Link href="/">
-                <Image src="/academy_logo.svg" width={150} height={40} alt="Academy Logo" />
-              </Link>
-              {/* <ThemeToggleButton hidden={isConnected} /> */}
-            </div>
-            <div className="mx-auto">
-              <TopBar menus={topbarNavMenus} />
-            </div>
+          <div className="flex justify-between">
+            <Link href="/">
+              <Image src="/academy_logo.svg" width={150} height={40} alt="Academy Logo" />
+            </Link>
+            {/* <ThemeToggleButton hidden={isConnected} /> */}
           </div>
         ) : (
           <div className="flex items-center justify-around gap-36 text-white lg:flex lg:justify-between lg:gap-5 lg:self-stretch">
-            <div className="lg:ml-8 lg:flex lg:basis-[0%] lg:flex-col lg:items-stretch">
+            <div className="lg:items-stretch9 lg:ml-8 lg:flex lg:basis-[0%] lg:flex-col">
               <BackButton />
             </div>
           </div>
         )}
       </div>
-      {pathname === "/" || pathname === "/tracks" || pathname === "/fundamentals" ? (
-        <div className="inline-flex gap-2">
-          {/* <ThemeToggleButton hidden={pathname !== "/" && isConnected ? false : true} /> */}
-          <ConnectButton />
-        </div>
-      ) : (
-        <>
-          <BackButton className="lg:hidden" />
-          <div className="hidden gap-2 lg:inline-flex">
+
```

Comment:
on a trackPage the topbar is present now and I believe this is not the desired behaviour.

![image](https://github.com/user-attachments/assets/9b0b177b-b92a-4475-ac08-18b94325c354)

also happening in the lesson page, it should be something on the topbar I guess.

![image](https://github.com/user-attachments/assets/d3babb40-fa3b-408e-b7a8-59c42b9a59d5)


Can you check it?

Reply:
The top bar is only visible on the `home page`, `/tracks`, and  `/fundamentals` pages only
<img width="1679" alt="Screenshot 2024-09-14 at 12 09 47 PM" src="https://github.com/user-attachments/assets/41104b5b-7fe8-45a0-9170-9e74e8549d75">
.